### ISSUE TITLE
Issue with 2.0.4 

### ISSUE THREAD

**Issue by jabesq** [home-assistant/homebridge-homeassistant#117]
When starting homebridge, this plugin produce the following error:

```
[2017-02-20 14:58:34] ERROR LOADING PLUGIN homebridge-homeassistant:
[2017-02-20 14:58:34] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
 at exports.runInThisContext (vm.js:53:16)
 at Module._compile (module.js:387:25)
 at Object.Module._extensions..js (module.js:422:10)
 at Module.load (module.js:357:32)
 at Function.Module._load (module.js:314:12)
 at Module.require (module.js:367:17)
 at require (internal/module.js:16:19)
 at Plugin.load (/usr/local/lib/node_modules/homebridge/lib/plugin.js:65:22)
 at Server.<anonymous> (/usr/local/lib/node_modules/homebridge/lib/server.js:132:14)
 at Array.forEach (native)
```

npm --version: 4.2.0

**Comment by rbflurry** [comment]
Seeing the same

**Comment by maddox** [comment]
Same.

**Comment by bl8rnr** [comment]
Had the same issue here:

To fix go to your homebridge-homeassistant install dir ie. /usr/local/lib/node_modules/homebridge and under accessories sub dir edit each of the .js files and add the following in a new line right at the top:

"use strict";

Also, add the above line to the index.js file under the install directory

**Comment by robbiet480** [comment]
Closed by #118 and fix deployed to npm as version 2.0.5. Sorry!
